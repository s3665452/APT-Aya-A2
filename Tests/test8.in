YBLBRUBLYRYRLRUYBLLLRRRYLUURULYBRBLUBRLYYURBYYULURULLBBUYULRYUBUBRULYRRUYURLBUYURYLBBURYLYYBRBLYBUBLB
Armin
Stuart
turn 5 L 3
turn 1 B 2
turn 3 R 2
turn 0 Y 3
turn 0 U 1
turn 4 Y 1
turn 0 L 4
turn 0 B 4
turn 0 R 5
turn 0 U 5

turn 1 R 3
turn 2 U 2
turn 3 L 4
turn 0 B 4
turn 4 R 5
turn 5 B 4
turn 0 R 5
turn 0 U 5
turn 0 B 1
turn 3 L 3
turn 0 Y 2