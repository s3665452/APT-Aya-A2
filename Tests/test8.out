Factories:
0: F
1: R Y L B
2: B U R Y
3: L Y Y B
4: R B L Y
5: B B L B

Score for Player Armin: 28
Mosaic for Armin:
1:         . || B . . U L
2:       . . || . B Y R U
3:     . B B || . L . . R
4:   . . . . || R U L . .
5: . Y Y Y Y || . R . . .
broken:

Score for Player Stuart: 23
Mosaic for Stuart:
1:         . || B Y . . .
2:       . . || L B . . U
3:     . . . || U . . Y R
4:   . . . . || . . L . Y
5: . . . . . || . . U . B
broken:
